### ISSUE TITLE
[8.0] Can't find an element in wizard

### ISSUE THREAD

**Issue by livingmine** [brain-tec/odoo-robot-framework#57]
I've noticed that in [odoo_9_0_EE.robot](https://github.com/brain-tec/odoo-robot-framework/blob/master/odoo_9_0_EE.robot#L164) and [odoo_10_0.robot](https://github.com/brain-tec/odoo-robot-framework/blob/master/odoo_10_0.robot#L160) there exist ButtonWizard (which i assume it is for clicking button in wizard) while there is no equivalent keyword for Odoo 8.0. I've been trying to trigger the click event using Selenium2Library ClickElement with proper xpath (copied directly from Copy Xpath from inspect mode) selector to achieve the same functionality but unfortunately the following error occured
```
ValueError: Element locator 'xpath=/html/body/div[5]/div/div/div[1]/button' did not match any elements.
```

Is it possible to achieve the same in Odoo 8.0?

**Comment by Jerther** [comment]
I think it's better to use the data-bt-* attributes added by the web_selenium module developed by brain-tec. 
If you're using odoo_robot-framework, my guess is that you already know about it but if not, it adds some attributes to your buttons and thus makes writing tests way easier.

Check it out: https://github.com/brain-tec/web_selenium

**Comment by livingmine** [comment]
@Jerther Yes, i've already knew and been using web_selenium. The thing is AFAIK, web_selenium does not seem to add the data-bt-* attributes to some elements and one of them is this "button in the wizard" element which i've been trying to interact with. I use data-bt-* attributes for most of other elements and in fact i'm using most of the keyword in [odoo_8_0.robot](https://github.com/brain-tec/odoo-robot-framework/blob/master/odoo_8_0.robot) without any problems except for this particular element. Am i missing something?

**Comment by Jerther** [comment]
Yeah sometimes the module fails to add attributes to buttons in a modal form. IIRC it may not be possible in a technical point of view. Here are some examples of what I do in this exceptional situation:

`Click Button    xpath=//div[@class='modal-footer']//button//span[.='Ok']/..`
`Click Button    xpath=//div[@class='modal-footer']//button/span[.='Cancel']/..`

These suppose there is only one modal shown. They work in Odoo 10 and should also work in Odoo 8.

If this still doesn't work, then make sure the modal form is loaded, actually visible and usable when you're calling the keywords.

**Comment by livingmine** [comment]
@Jerther Thanks for the tips! Anyway, i was wrong about this particular element i've described above. The data-bt-* attribute exists, however it seems that the selector can not locate the element, possibly due to its appearance is inside the modal. I'll try your suggestions first then reporting back to this thread.
